stable/mongodb-replicaset/ci/tls-values.yaml

```diff
@@ -0,0 +1,10 @@
+tls:
+  # Enable or disable MongoDB TLS support
+  enabled: true
+  # Please generate your own TLS CA by generating it via:
```

Comment:
You can remove these comments. This is a test file with static values for continuous integration, it's not for use by users.